src/main/java/com/codeborne/selenide/SelenideConfig.java

```diff
@@ -23,22 +23,17 @@
 
   private String baseUrl = System.getProperty("selenide.baseUrl", "http://localhost:8080");
   private long timeout = Long.parseLong(System.getProperty("selenide.timeout", "4000"));
-  private long collectionsTimeout = Long.parseLong(System.getProperty("selenide.collectionsTimeout", "6000"));
-  private long pollingInterval = Long.parseLong(System.getProperty("selenide.pollingInterval", "100"));
-  private long collectionsPollingInterval = Long.parseLong(System.getProperty("selenide.collectionsPollingInterval", "200"));
+  private long pollingInterval = Long.parseLong(System.getProperty("selenide.pollingInterval", "200"));
```

Comment:
Are you really sure we want to increase default `pollingInterval` from 100ms to 200ms? Why?
As to me, 100 was a reasonable default. Probably we could make it even fewer.

Reply:
This value going from collections polling interval.
With `100` value related test going to be unstable.
There are two options:
1. Set to `100` and revise tests
2. Leave `200`

Reply:
@rosolko ok, we can leave 200, and later investigate why some tests fail with value 100.